samples/storecdm/README.md

```diff
@@ -0,0 +1,43 @@
+# clearkeyStoreCDM
```

Comment:
Need to update build yaml script to build the sample in the repo. It's located in root folder